services/ir/docker-compose.yml

```diff
@@ -15,12 +15,14 @@ services:
         ipv4_address: ${IPV4_PREFIX}.61
     stop_signal: SIGKILL
     volumes:
-      - ./01.key:/01.key
+      - ./node-wallet.json:/node-wallet.json
```

Comment:
Maybe simplify filename to `wallet.json`? Or as for keys `wallet01.json`.

Reply:
Created symlink with `wallet01.json` name.